### PR TITLE
Pre RPM packaging changes

### PR DESCRIPTION
**Pull request by jessp01**
- placeholders for paths in init script
- added forever to list of deps
- exec bit on for init script

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -100,8 +94,13 @@
 		  	<replacefilter token="@CLOUD_SECRET@" property="user_input.CLOUD_SECRET"/>
 			<replacefilter token="@CLOUD_SHARED_BASE_PATH@" property="user_input.CLOUD_SHARED_BASE_PATH"/>
 			<replacefilter token="@LOG_DIR@" property="user_input.LOG_DIR"/>
+			<replacefilter token="@PLAY_SERVER_BASE_DIR@" property="user_input.PLAY_SERVER_BASE_DIR"/>
 			<replacefilter token="@BIN_DIR@" value="${bin.dir}"/>
 		</replace>
+		<replace file="${basedir}/bin/play-server.sh" value="@TOKEN_VALU_NOT_FOUND@" propertyFile="${configFilePath}">
+			<replacefilter token="@LOG_DIR@" property="user_input.LOG_DIR"/>
+			<replacefilter token="@PLAY_SERVER_BASE_DIR@" property="user_input.PLAY_SERVER_BASE_DIR"/>
```
Comment (yuly-roberman) [inline build.xml]: we don't need PLAY_SERVER_BASE_DIR in user_input.ini, please change this line to:
<replacefilter token="@PLAY_SERVER_BASE_DIR@" value="${basedir}"/>

File: build.xml [hunk 2]
```diff
@@ -100,8 +94,13 @@
 		  	<replacefilter token="@CLOUD_SECRET@" property="user_input.CLOUD_SECRET"/>
 			<replacefilter token="@CLOUD_SHARED_BASE_PATH@" property="user_input.CLOUD_SHARED_BASE_PATH"/>
 			<replacefilter token="@LOG_DIR@" property="user_input.LOG_DIR"/>
+			<replacefilter token="@PLAY_SERVER_BASE_DIR@" property="user_input.PLAY_SERVER_BASE_DIR"/>
```
Comment (yuly-roberman) [inline build.xml]: we don't need PLAY_SERVER_BASE_DIR in user_input.ini, please change this line to:<replacefilter token="@PLAY_SERVER_BASE_DIR@" value="${basedir}"/>

File: config/user_input.ini.template [hunk 3]
```diff
@@ -3,4 +3,5 @@ user_input.PLAY_PARTNER_ADMIN_SECRET = @PLAY_PARTNER_ADMIN_SECRET@
 user_input.CLOUD_HOSTNAME = @CLOUD_HOSTNAME@
 user_input.CLOUD_SECRET = @CLOUD_SECRET@
 user_input.CLOUD_SHARED_BASE_PATH = @CLOUD_SHARED_BASE_PATH@
-user_input.LOG_DIR = @LOG_DIR@
\ No newline at end of file
+user_input.LOG_DIR = @LOG_DIR@
+user_input.PLAY_SERVER_BASE_DIR = @PLAY_SERVER_BASE_DIR@
```
Comment (yuly-roberman) [inline config/user_input.ini.template]: please remove PLAY_SERVER_BASE_DIR  definition

**Comment by jessp01** [comment]
Pushed new build.xml

**Pull request merged**